lib/src/widget.dart

```diff
@@ -43,6 +44,9 @@ class AdvancedCalendar extends StatefulWidget {
   /// List of points for the week and mounth
   final List<DateTime>? events;
 
+  /// The first day of the week starts
+  final DateTime? startWeekDay;
```

Comment:
startWeekDay - should be a number of a week day or enum, it shouldn't be a concrete date